apps/site/components/Common/Searchbox/Search/index.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import { MagnifyingGlassIcon } from '@heroicons/react/24/solid';
+import { SearchInput } from '@orama/ui/components';
+import { useTranslations } from 'next-intl';
+import { type FC, type PropsWithChildren } from 'react';
```

Comment:
small nit: this can be a type import

```suggestion
import type { FC, PropsWithChildren } from 'react';
```